trace/trace.go

```diff
@@ -106,6 +115,14 @@ type Trace struct {
 	traceItems []traceItem
 }
 
+func (t *Trace) rlock() {
+	t.lock.RLock()
+}
+
+func (t *Trace) runlock() {
```

Comment:
my eye/brain reads this as "run lock" :(

Reply:
`rUnlock`?

Reply:
then you would have to use `rLock`?

Reply:
Yes.

Reply:
Updated accordingly.